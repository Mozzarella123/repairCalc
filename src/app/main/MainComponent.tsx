import React from "react";
import classnames from 'classnames';
import { RoomListContainer } from "./rooms/RoomListContainer";
import { AddProjectFormContainer } from "./AddProjectFormContainer";

interface Props {
    project?: any
}

export const MainComponent = ({ project }: Props) => (
    (project) ?
        <RoomListContainer />
        :
        <AddProjectFormContainer />
)

export default MainComponent;