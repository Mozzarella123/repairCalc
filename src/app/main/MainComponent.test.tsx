import { shallow, mount } from 'enzyme';
import toJSON from 'enzyme-to-json'
import * as React from "react";
import {MainComponent}  from './MainComponent'

jest.mock('./rooms/RoomListContainer', () => "rooms")
jest.mock('./AddProjectFormContainer', () => "form")

describe("MainComponent UI test", () => {
    it('renders', () => {
    const wrapper = shallow(<div>
      <h1>Hello, Enzyme!</h1>
    </div>)
    expect(wrapper.find('h1').html()).toMatch(/Hello, Enzyme/)
  })
    it("Renders Correctly", () => {
        expect(shallow(<MainComponent />)).toMatchSnapshot()
    })
})