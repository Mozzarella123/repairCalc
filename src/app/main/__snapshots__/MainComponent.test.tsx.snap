// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainComponent UI test Renders Correctly 1`] = `<Component />`;
